examples/28-wireframe/wireframe.cpp

```diff
@@ -394,6 +407,35 @@ class ExampleWireframe : public entry::AppI
 			ImGui::RadioButton("Wireframe", &m_drawMode, 1);
 			ImGui::RadioButton("Shaded", &m_drawMode, 2);
 
+			static bool wfBaryCoordEnabled = false;
+			if (!m_hasBaryCoord)
+			{
+				ImGui::BeginDisabled();
+			}
+
+			ImGui::Separator();
+
+			ImGui::Checkbox("Shader barycentric coords", &wfBaryCoordEnabled);
+
```

Comment:
You could do:
`wfBaryCoordEnabled &= m_hasBaryCoord;`

And then you don't need to test `m_hasBaryCoord` again.

Reply:
I did that to gray out the feature if it is not available, or do you mean having it on by default if the feature is there?